clientupdate/clientupdate.go

```diff
@@ -833,14 +833,25 @@ func (up *Updater) installMSI(msi string) error {
 	return err
 }
 
+// cleanupOldDownloads removes all files matching glob (see filepath.Glob).
+// Only regular files are removed, so the glob must match specific files and
+// not directories.
 func (up *Updater) cleanupOldDownloads(glob string) {
 	matches, err := filepath.Glob(glob)
 	if err != nil {
 		up.Logf("cleaning up old downloads: %v", err)
 		return
 	}
 	for _, m := range matches {
-		if err := os.RemoveAll(m); err != nil {
+		s, err := os.Stat(m)
```

Comment:
I think we want LStat, so it doesn't follow a symlink

Reply:
:facepalm: